sort: add --general-numeric-sort example

hntee opened the pull request:
<!-- Thank you for sending a PR! -->
<!-- Please perform the following checks and mark all the boxes accordingly. -->
<!-- You can remove the checklist items that don't apply to your PR. -->

- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/tldr-pages/tldr?pullRequest=4325) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/tldr-pages/tldr?pullRequest=4325) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/tldr-pages/tldr?pullRequest=4325) it.</sub>

navarroaxel commented on pages/common/sort.md:
```diff
@@ -29,3 +29,7 @@
+
+- Sort numbers with exponents:
+
+`sort -g {{filename}}`
```
we prefer the long argument:
```suggestion
`sort --general-numeric-sort {{path/to/file}}`
```
await confirmation before apply

navarroaxel reviewed (commented):
Hey @hntee thanks for your contribution! I think this page is some outdated respect the style guide.

Could you add the link to the documentation in the page description?
```
> Sort lines of text files.
> More information: <https://www.gnu.org/software/coreutils/manual/html_node/sort-invocation.html>.
```

hntee commented:
Hi @navarroaxel , thanks for your reply.
I have changed the parameter to the longer one and added the link.

Another question: since the above descriptions are all `{{filename}}`, is it better to modify all of these to `{{path/to/file}}`?

navarroaxel commented:
> Hi @navarroaxel , thanks for your reply.
> I have changed the parameter to the longer one and added the link.

@hntee thanks!

> Another question: since the above descriptions are all `{{filename}}`, is it better to modify all of these to `{{path/to/file}}`?

I had that in mind, and also fix some arguments like `-r` to `--reverse` but I want the confirmation from @sbrl before apply those changes. Sometime there are exceptions for the rule.

sbrl reviewed (approved):
Thanks, @hntee!

Yeah since we're here, if you wouldn't mind adjusting all the other `{{filename}}` instances to `{{path/to/filename}}` that would be great!

Generally it's only pages like `diff` for example that are exceptions.

hntee commented:
Hi @navarroaxel @sbrl , I have modified all the {{filename}} instances to {{path/to/filename}}

sbrl commented:
Thanks, @hntee :smiley_cat:
